ops/terraform/modules/resources/bfd_pipeline/main.tf

```diff
@@ -134,6 +134,35 @@ resource "aws_security_group_rule" "allow_db_primary_access" {
   source_security_group_id = aws_security_group.app.id # The EC2 instance for the BFD Pipeline app.
 }
 
+# NACL to manage communication between the BFD and RDA environments
+# By default this NACL won't add any additonal security, however if for any reason
+# we need to shut off communication between the two VPCs, one could simply flip the
+# action(s) defined in these rules from "ALLOW" to "DENY"
+resource "aws_network_acl" "rda" {
+  count = var.mpm_rda_cidr_block ? 1 : 0
```

Comment:
I don't believe this will work because the ternary expression must eval to a bool.  You could try `count = var.mpm_rda_cidr_block != "" ? 1 : 0`

Reply:
makes sense, I update this